src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionService.java

```diff
@@ -91,6 +91,41 @@ public interface SnowflakeConnectionService {
    */
   boolean isTableCompatible(String tableName);
 
+  /**
+   * Check whether the user has the role privilge to do schema evolution and whether the schema
+   * evolution option is enabled on the table
```

Comment:
Is this available on mixed tables?

Reply:
This is new for me. 

How can I create a mixed table to have some experiments?

Reply:
sounds good! 
I just wanted to make sure if this code would work if we disabled schema evolution on this streaming table. But we can add that as an action item to verify before we announce.